changelog/5153.trivial.rst

```diff
@@ -0,0 +1,2 @@
+Remove extra characters in the shown command line prompts in the test discovery
+documentation.
```

Comment:
btw: I do not think we need a changelog for this.

Reply:
me neither. creating this was pain via github.

Reply:
https://github.com/pytest-dev/pytest/blob/master/changelog/README.rst

> `trivial`: fixing a small typo or internal change that might be noteworthy.

Note sure if the "might be noteworthy" part applies to fixing typos, but I think this list should be updated to say not to create this just for a trivial typo fix.

Reply:
Oh excellent point, thanks for pointing that out! We should update that templates to mean that trivial is meant to actual code or dependency changes, but not to small documentation changes in general. 👍